server/auth/handlers/handle_profile.go

```diff
@@ -25,6 +32,10 @@ func (h *AuthHandlers) profileForm(req *request.AuthReq) error {
 		}
 	}
 
+	if u.Meta.AvatarID != 0 {
+		avatarUrl = fmt.Sprintf("/api/system/attachment/avatar/%d/original/%s", u.Meta.AvatarID, types.AttachmentKindAvatar)
```

Comment:
The `/api` prefix can in some configurations be omitted or totally different, see more: https://github.com/cortezaproject/corteza/blob/2023.3.x/server/app/servers.go#L71